src/js/dataMapping/search/awardType.js

```diff
@@ -10,6 +10,14 @@ export const awardTypeCodes = {
     'B': 'Purchase Orders (PO)',
     'C': 'Delivery Orders (DO)',
     'D': 'Definitive Contracts',
+    'IDV_A': 'GWAC',
+    // IDV_B: <to be determined>
```

Comment:
Is this still TBD?

Reply:
It seems that IDV_B may be a category, in which case, the variables `IDV_B_A`, `IDV_B_B`, `IDV_B_C` would be nested under it and we'd need a name for the category.